crates/cli/src/commands/install.rs

```diff
@@ -218,6 +219,19 @@ mod tests {
     use pretty_assertions::assert_eq;
     use tempfile::tempdir;
 
+    // Helper function to check if a path is a symlink or junction
+    fn is_symlink_or_junction(path: std::path::PathBuf) -> Result<bool> {
+        #[cfg(windows)]
+        {
+            junction::exists(&path)
+        }
+
+        #[cfg(not(windows))]
+        {
+            Ok(path.is_symlink())
+        }
```

Comment:
`return` statements use less lines:

```suggestion
        #[cfg(unix)]
        return Ok(path.is_symlink());
        #[cfg(windows)]
        return junction::exists(&path);
```